### ISSUE TITLE
Server Issue

### ISSUE THREAD

**Issue by ihabhamad** [dOpensource/dsiprouter#243]
Server Issue
Detailed Information: HTTPConnectionPool(host='127.0.0.1', port=5060): Max retries exceeded with url: /api/kamailio (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fe40683ab70>: Failed to establish a new connection: [Errno 111] Connection refused',))
![image](https://user-images.githubusercontent.com/16678100/87145631-45d86c00-c2b2-11ea-85f5-3c8eb024c8a2.png)
server version 0.62

**Comment by ihabhamad** [comment]
![image](https://user-images.githubusercontent.com/16678100/87145568-2fcaab80-c2b2-11ea-80b2-de2ebe25801e.png)

**Comment by ihabhamad** [comment]
/dsiprouter.sh configurekam
ERROR 1064 (42000) at line 1: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'IF NOT EXISTS 'kamailio'@'localhost' IDENTIFIED BY 'HrSFbkCDTONoOzlXRovr7SEYaxcN' at line 1

**Comment by mackhendricks** [comment]
Not sure why this is happening.  If the system is still up I would like to check it out.

**Comment by amen160** [comment]
me too
Detailed Information: HTTPConnectionPool(host='127.0.0.1', port=5060): Max retries exceeded with url: /api/kamailio (Caused by NewConnectionError('<requests.packages.urllib3.connection.HTTPConnection object at 0x7fa1ca688978>: Failed to establish a new connection: [Errno 111] Connection refused',))

**Comment by mackhendricks** [comment]
I've seen this as well.  Does this happen after  you attempt to do a "Reload".  That api tries to reload kamailio.  Can you check if Kamailio is still running?  Did you attempt to change the Kamailio database password? or did this error just pop up?

**Comment by amen160** [comment]
<img width="1400" alt="Screen Shot 2020-07-22 at 15 25 43" src="https://user-images.githubusercontent.com/6201829/88153490-af9c3280-cc2f-11ea-9d75-bb0ace0295db.png">

**Comment by amen160** [comment]
@mackhendricks Kamailio not run.
<img width="944" alt="Screen Shot 2020-07-22 at 15 27 28" src="https://user-images.githubusercontent.com/6201829/88153634-dbb7b380-cc2f-11ea-89fc-31e4bdfa3ab2.png">

**Comment by amen160** [comment]
why we did not use docker for dsiprouter?

**Comment by mackhendricks** [comment]
What cloud provider and and OS are you using?

Looks like dSIP is having a problem updating your network configuration

We didn’t use docker for Kamailio because we wanted the highest level of performance

**Comment by amen160** [comment]
I use Centos 7 on droplet digitalocean.
the same with vultr.com & google
I have tried installing your project many times but never been successful. :(.

**Comment by mackhendricks** [comment]
I would not recommend using Centos 7 because we haven't kept that installation script up to date.  

I would use Debian 9

**Comment by ypokg** [comment]
Hello there. We're having the same issue on Centos7

**Comment by mackhendricks** [comment]
Please install branch v0.64

**Comment by ypokg** [comment]
Hi @mackhendricks, thanks for your response.

We're still experiencing this issue. Here's what I did.

I stopped dSIP and uninstalled it, then ran:

```
git checkout v0.64
./dsiprouter.sh install -all -servernat
./dsiprouter.sh start

```
Below is the git status
```

[root@localhost dsiprouter]# git status
# On branch v0.64
# Changes not staged for commit:
#   (use "git add <file>..." to update what will be committed)
#   (use "git checkout -- <file>..." to discard changes in working directory)
#
#       modified:   gui/modules/fusionpbx/certs/cert.key
#       modified:   gui/modules/fusionpbx/certs/cert_combined.crt
#       modified:   gui/settings.py
#
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#       .rtpengineinstalled
#       docs/build/
no changes added to commit (use "git add" and/or "git commit -a")
```

Here's the full error we're getting:

**Server Issue**
Detailed Information: HTTPConnectionPool(host='127.0.0.1', port=5060): Max retries exceeded with url: /api/kamailio (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f1b4a3e87b8>: Failed to establish a new connection: [Errno 111] Connection refused',))

What am I missing?

Let me know if you need anything further

Thank you very much

**Comment by reqlez** [comment]
The team is finishing up some cleanups for the install script.  I think if you wait a bit it will be fixed?  I can confirm most things are now fixed but there are a few more bugs that I saw that need a bit more work.

**Comment by RichSosa28** [comment]
This is no longer an issue. It has been fixed in our v0.64 branch which is now released and merged into our master branch.
